code/oot.ld

```diff
@@ -8,6 +8,14 @@ SECTIONS
 		*(.patch_loader)
 	}
 
+	.patch_SleepQueryCallback 0x3FD6C4 : {
+		*(.patch_SleepQueryCallback)
+	}
+
+	.patch_AwakeCallback 0x3FD43C : {
+		*(.patch_AwakeCallback)
+	}
+
```

Comment:
Could you move these to be sorted by address? Just for consistency with the rest of the file.

Reply:
I somehow managed to completely not notice the patches were ordered by address in that file! Fixed in the last push.